src/xcode/ENA/ENA/Source/Services/Risk/Provider/__tests__/RiskProviderTests.swift

```diff
@@ -340,19 +366,15 @@ final class RiskProviderTests: XCTestCase {
 		}
 
 		waitForExpectations(timeout: .long)
-
 		XCTAssertFalse(store.shouldShowRiskStatusLoweredAlert)
 	}
 
 	func testShouldShowRiskStatusLoweredAlertInitiallyTrueKeepsValueWhenRiskStatusStaysLow() throws {
 		let store = MockTestStore()
 		store.shouldShowRiskStatusLoweredAlert = true
-
 		let riskProvider = try riskProviderChangingRiskLevel(from: .low, to: .low, store: store)
-
```

Comment:
Hmm would say that removing all those newlines was not really needed, I might even has been easier to read with them.

Reply:
Added line breaks again.